beets/util/__init__.py

```diff
@@ -16,20 +16,20 @@
 
 import errno
 import fnmatch
-import functools
```

Comment:
This is still required.

Reply:
Yup I accidentally removed it when I reversed my `cached_property` import, assuming that nothing else needs this import.